src/sentry/incidents/subscription_processor.py

```diff
@@ -673,32 +679,29 @@ def anomaly_has_confidence(self, anomaly) -> bool:
         return anomaly_type != AnomalyType.NO_DATA.value
 
     def get_anomaly_data_from_seer(self, aggregation_value: float | None):
+        anomaly_detection_config = AnomalyDetectionConfig(
+            time_period=int(self.alert_rule.snuba_query.time_window / 60),
```

Comment:
maybe could just do ` // 60` instead?

Reply:
what's the difference? does that automatically make it an integer?